app/gui/view/src/project.rs

```diff
@@ -163,7 +163,7 @@ ensogl::define_endpoints! {
         // TODO[MM]: this should not contain the group entry id as that is component browser
         // specific. It should be refactored to be an implementation detail of the component
         // browser.
-        editing_committed              (NodeId, Option<component_list_panel::grid::GroupEntryId>),
+        editing_committed              (NodeId, Option<component_list_panel::grid::EntryId>),
```

Comment:
Is the `TODO` above still needed?

Reply:
I think yes @MichaelMauderer can you confirm? It was added when implementing AI prompt.